core/src/main/kotlin/org/evomaster/core/problem/graphQl/Main.kt

```diff
@@ -0,0 +1,78 @@
+import com.google.gson.Gson
+import java.io.File
+import java.io.InputStream
+
+class Main {
+
```

Comment:
it is fine to have some classes to play with dubugging. However:
1) they should not be under `src/main/kotlin`, but rather under `src/test/kotlin` (even if they are not test cases)
2) use  a more meaningful name, to avoid confusing this class with the other `Main` which is the entry point for the whole application, eg  something like`GraphQLPetStoreCheck`

Reply:
done